Makefile

```diff
@@ -1,19 +1,16 @@
-all: bounties sunodo-sdk-image image
+all: bounties test-image image
 
 image:
 	sunodo build
 
 bounties:
 	$(MAKE) -C tests/bounties
 
-sunodo-sdk-image: linux.bin
```

Comment:
Remember to remove the mention to this `sunodo-sdk-image` from README.md.

Reply:
I've removed all mentions to the `sunodo-sdk-image` Makefile target.